lib-python/littleboxes/xword.py

```diff
@@ -1,22 +1,24 @@
 import logging
 import puz
 
+
```

Comment:
autopep8 is great! I've added it to my editor and will fix up the existing violations